packages/@aws-cdk/aws-sns-subscriptions/test/subs.test.ts

```diff
@@ -1927,6 +1927,53 @@ test('with filter policy', () => {
   });
 });
 
+test('with filter policy scope MessageBody', () => {
+  const fction = new lambda.Function(stack, 'MyFunc', {
```

Comment:
```suggestion
  const func = new lambda.Function(stack, 'MyFunc', {
```
Just to match the other changes you've already made.